util/load_profile.py

```diff
@@ -0,0 +1,60 @@
+import csv
```

Comment:
seems to be duplicate of util/parse_standard_load_profile.py

Reply:
Right, I will delete this file!